src/main/content/_assets/js/collections.js

```diff
@@ -1,82 +1,155 @@
-$(document).ready(function() {
-    fetchAllInstances()
-        .then(setInstanceSelections)
-        .then(handleInitialCLIAuth)
-        .then(handleCollectionsRequests)
-    //TODO then select/fetch current instance from url param for the dropdown
-
+$(document).ready(function () {
+    handleInstancesRequests();
     $("#instance-accordion li").on("click", e => {
         // prevent carbon from doing its normal thing with the accordion
         e.stopPropagation();
-        
+
         let newName = handleInstanceSelection(e.target);
         fetchAnInstance(newName)
             .then(updateInstanceView);
     });
+
+    $("#sync-collections-icon").on("click", (e) => {
+        if (e.target.getAttribute("class") == "icon-active") {
+            let instanceName = $("#instance-accordion").find(".bx--accordion__title").text();
+            hideAndEmptyTable();
+            syncColletions(instanceName);
+        }
+    });
+
+    $("#collection-table-body").on("click", ".deactivate-collection-icon", (e) => {
+        let collectionName = e.currentTarget.getAttribute("collection-name");
+        $("#modal-collection-name").text(collectionName);
+    });
+
+    $("#modal-confirm-deactivation").on("click", () => {
+        let collectionName = $("#modal-collection-name").text();
+        deactivateCollection(collectionName);
```

Comment:
do collections from the cli server come with an ID of sorts? Or do we have to rely on the name?